resources/vault_install_binary.rb

```diff
@@ -23,11 +23,13 @@
 default_action :install
 
 action :install do
-  directory new_resource.extract_to do
-    mode '0755'
-    owner new_resource.user
-    group new_resource.group
-    recursive true
+  [new_resource.extract_to, ::File.join(new_resource.extract_to, new_resource.version)].each do |dir|
+    directory dir do
+      mode '0755'
+      owner new_resource.user
+      group new_resource.group
+      recursive true
+    end
```

Comment:
Good question. Let's try and find a solution with the group. I'm collecting luggage and heading home to sleep. Will find some time tomorrow to check back in after I get up and figure out home.